tiling.py

```diff
@@ -458,8 +821,46 @@ def fundamental_domain_transforms(self):
 
     @property
     def center(self):
-        return 0
+        return self._center
+
 
+# class DihedralKernel(object):
+#     def __init__(self, group):
+#         self._group = group
+#
+#     def __str__(self):
+#         return "Dihedral Kernel of order %d" % self._group.n
+#
+#     def adjacent_kernels(self, overlap):
+#         return []
+#
+#     @property
+#     def fundamental_domains(self):
+#         for k in range(len(self._group.dihedral_subgroup)):
+#             transform = self._group.dihedral_subgroup[k]
+#             prism = shapes.Prism(self._group.height, *self._group.fd_vertices)
+#             prism.transform(transform)
+#             yield (0, k), transform, prism
+#
+#     @property
+#     def translational_fundamental_domains(self):
+#         prism = shapes.Prism(self._group.height, *self._group.translational_fd_vertices)
+#         yield 0, np.eye(4), prism
+#
+#     @property
+#     def translational_fundamental_domain_transforms(self):
+#         for _, tx, _ in self.translational_fundamental_domains:
+#             yield tx
+#
+#     @property
+#     def fundamental_domain_transforms(self):
+#         for _, tx, _ in self.fundamental_domains:
+#             yield tx
+#
+#     @property
+#     def center(self):
+#         return 0
+#
```

Comment:
I kind of feel this part of your old codes was not useful anymore. But not sure if you saved them for future development. So I just commented them.